django_token_bucket/migrations/0003_auto_20200111_1625.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 2.1.15 on 2020-01-11 15:25
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.db import migrations
+
+
+def forwards(apps, schema_editor):
+    token_bucket_model = apps.get_model('django_token_bucket', 'TokenBucket')
+    content_type_model = apps.get_model('contenttypes', 'ContentType')
+    for row in token_bucket_model.objects.all():
+        if not row.content_type:
+            row.content_type = content_type_model.objects.get_for_model(get_user_model())
```

Comment:
To be safe, migrations should explicitly name the database to use (here `using(db_alias)`, where `db_alias` is  `schema_editor.connection.alias`).